e2e/test/MessageReceiveE2ETests.cs

```diff
@@ -153,7 +160,7 @@ private async Task ReceiveSingleMessage(TestDeviceType type, Client.TransportTyp
                     transport == Client.TransportType.Mqtt_WebSocket_Only)
                 {
                     // Dummy ReceiveAsync to ensure mqtt subscription registration before SendAsync() is called on service client.
-                    await deviceClient.ReceiveAsync(TimeSpan.FromSeconds(1)).ConfigureAwait(false);
+                    await deviceClient.ReceiveAsync(TimeSpan.FromSeconds(10)).ConfigureAwait(false);
```

Comment:
Why does this need to change? (Or exist in the first place?).

Reply:
1 s may be too short for receiveAsync - subscription, receive and all these together. As we are checking remaining time later with new changes, it seemed right to change this to a bigger number so it does not fail in e2e tests.